packages/theme-chalk/src/table-v2.scss

```diff
@@ -110,6 +110,10 @@
   @include e('header') {
     position: relative;
     overflow: hidden;
+
+    .#{$namespace}-checkbox {
+      z-index: 0;
+    }
```

Comment:
Maybe we could add `z-index: 1;` on line 89? 🤔

Reply:
@btea Yes, this is compatible with components other than checkbox, I'll add

Reply:
[fixed on left and right exaples](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZGl2IHN0eWxlPVwiaGVpZ2h0OiA0MDBweFwiPlxuICAgIDxlbC1hdXRvLXJlc2l6ZXI+XG4gICAgICA8dGVtcGxhdGUgI2RlZmF1bHQ9XCJ7IGhlaWdodCwgd2lkdGggfVwiPlxuICAgICAgICA8ZWwtdGFibGUtdjJcbiAgICAgICAgICA6Y29sdW1ucz1cImNvbHVtbnNcIlxuICAgICAgICAgIDpkYXRhPVwiZGF0YVwiXG4gICAgICAgICAgOndpZHRoPVwid2lkdGhcIlxuICAgICAgICAgIDpoZWlnaHQ9XCJoZWlnaHRcIlxuICAgICAgICAgIGZpeGVkXG4gICAgICAgIC8+XG4gICAgICA8L3RlbXBsYXRlPlxuICAgIDwvZWwtYXV0by1yZXNpemVyPlxuICA8L2Rpdj5cbjwvdGVtcGxhdGU+XG5cbjxzY3JpcHQgbGFuZz1cInRzeFwiIHNldHVwPlxuaW1wb3J0IHsgcmVmLCB1bnJlZiB9IGZyb20gJ3Z1ZSdcbmltcG9ydCB7IEVsQ2hlY2tib3gsIFRhYmxlVjJGaXhlZERpciB9IGZyb20gJ2VsZW1lbnQtcGx1cydcblxuaW1wb3J0IHR5cGUgeyBGdW5jdGlvbmFsQ29tcG9uZW50IH0gZnJvbSAndnVlJ1xuaW1wb3J0IHR5cGUgeyBDaGVja2JveFZhbHVlVHlwZSwgQ29sdW1uIH0gZnJvbSAnZWxlbWVudC1wbHVzJ1xuXG50eXBlIFNlbGVjdGlvbkNlbGxQcm9wcyA9IHtcbiAgdmFsdWU6IGJvb2xlYW5cbiAgaW50ZXJtZWRpYXRlPzogYm9vbGVhblxuICBvbkNoYW5nZTogKHZhbHVlOiBDaGVja2JveFZhbHVlVHlwZSkgPT4gdm9pZFxufVxuXG5jb25zdCBTZWxlY3Rpb25DZWxsOiBGdW5jdGlvbmFsQ29tcG9uZW50PFNlbGVjdGlvbkNlbGxQcm9wcz4gPSAoe1xuICB2YWx1ZSxcbiAgaW50ZXJtZWRpYXRlID0gZmFsc2UsXG4gIG9uQ2hhbmdlLFxufSkgPT4ge1xuICByZXR1cm4gKFxuICAgIDxFbENoZWNrYm94XG4gICAgICBvbkNoYW5nZT17b25DaGFuZ2V9XG4gICAgICBtb2RlbFZhbHVlPXt2YWx1ZX1cbiAgICAgIGluZGV0ZXJtaW5hdGU9e2ludGVybWVkaWF0ZX1cbiAgICAvPlxuICApXG59XG5cbmNvbnN0IGdlbmVyYXRlQ29sdW1ucyA9IChsZW5ndGggPSAxMCwgcHJlZml4ID0gJ2NvbHVtbi0nLCBwcm9wcz86IGFueSkgPT5cbiAgQXJyYXkuZnJvbSh7IGxlbmd0aCB9KS5tYXAoKF8sIGNvbHVtbkluZGV4KSA9PiAoe1xuICAgIC4uLnByb3BzLFxuICAgIGtleTogYCR7cHJlZml4fSR7Y29sdW1uSW5kZXh9YCxcbiAgICBkYXRhS2V5OiBgJHtwcmVmaXh9JHtjb2x1bW5JbmRleH1gLFxuICAgIHRpdGxlOiBgQ29sdW1uICR7Y29sdW1uSW5kZXh9YCxcbiAgICBmaXhlZDogY29sdW1uSW5kZXggPT0gMCB8fCBjb2x1bW5JbmRleCA9PSBsZW5ndGggLTEgPyB0cnVlIDogZmFsc2UsXG4gICAgd2lkdGg6IDE1MCxcbiAgfSkpXG5cbmNvbnN0IGdlbmVyYXRlRGF0YSA9IChcbiAgY29sdW1uczogUmV0dXJuVHlwZTx0eXBlb2YgZ2VuZXJhdGVDb2x1bW5zPixcbiAgbGVuZ3RoID0gMjAwLFxuICBwcmVmaXggPSAncm93LSdcbikgPT5cbiAgQXJyYXkuZnJvbSh7IGxlbmd0aCB9KS5tYXAoKF8sIHJvd0luZGV4KSA9PiB7XG4gICAgcmV0dXJuIGNvbHVtbnMucmVkdWNlKFxuICAgICAgKHJvd0RhdGEsIGNvbHVtbiwgY29sdW1uSW5kZXgpID0+IHtcbiAgICAgICAgcm93RGF0YVtjb2x1bW4uZGF0YUtleV0gPSBgUm93ICR7cm93SW5kZXh9IC0gQ29sICR7Y29sdW1uSW5kZXh9YFxuICAgICAgICByZXR1cm4gcm93RGF0YVxuICAgICAgfSxcbiAgICAgIHtcbiAgICAgICAgaWQ6IGAke3ByZWZpeH0ke3Jvd0luZGV4fWAsXG4gICAgICAgIGNoZWNrZWQ6IGZhbHNlLFxuICAgICAgICBwYXJlbnRJZDogbnVsbCxcbiAgICAgIH1cbiAgICApXG4gIH0pXG5cbmNvbnN0IGNvbHVtbnM6IENvbHVtbjxhbnk+W10gPSBnZW5lcmF0ZUNvbHVtbnMoMTApXG5jb2x1bW5zW2NvbHVtbnMubGVuZ3RoIC0gMV0uZml4ZWQgPSBUYWJsZVYyRml4ZWREaXIuUklHSFRcbmNvbHVtbnMudW5zaGlmdCh7XG4gIGtleTogJ3NlbGVjdGlvbicsXG4gIHdpZHRoOiA1MCxcbiAgXG4gIGNlbGxSZW5kZXJlcjogKHsgcm93RGF0YSB9KSA9PiB7XG4gICAgY29uc3Qgb25DaGFuZ2UgPSAodmFsdWU6IENoZWNrYm94VmFsdWVUeXBlKSA9PiAocm93RGF0YS5jaGVja2VkID0gdmFsdWUpXG4gICAgcmV0dXJuIDxTZWxlY3Rpb25DZWxsIHZhbHVlPXtyb3dEYXRhLmNoZWNrZWR9IG9uQ2hhbmdlPXtvbkNoYW5nZX0gLz5cbiAgfSxcblxuICBoZWFkZXJDZWxsUmVuZGVyZXI6ICgpID0+IHtcbiAgICBjb25zdCBfZGF0YSA9IHVucmVmKGRhdGEpXG4gICAgY29uc3Qgb25DaGFuZ2UgPSAodmFsdWU6IENoZWNrYm94VmFsdWVUeXBlKSA9PlxuICAgICAgKGRhdGEudmFsdWUgPSBfZGF0YS5tYXAoKHJvdykgPT4ge1xuICAgICAgICByb3cuY2hlY2tlZCA9IHZhbHVlXG4gICAgICAgIHJldHVybiByb3dcbiAgICAgIH0pKVxuICAgIGNvbnN0IGFsbFNlbGVjdGVkID0gX2RhdGEuZXZlcnkoKHJvdykgPT4gcm93LmNoZWNrZWQpXG4gICAgY29uc3QgY29udGFpbnNDaGVja2VkID0gX2RhdGEuc29tZSgocm93KSA9PiByb3cuY2hlY2tlZClcblxuICAgIHJldHVybiAoXG4gICAgICA8U2VsZWN0aW9uQ2VsbFxuICAgICAgICB2YWx1ZT17YWxsU2VsZWN0ZWR9XG4gICAgICAgIGludGVybWVkaWF0ZT17Y29udGFpbnNDaGVja2VkICYmICFhbGxTZWxlY3RlZH1cbiAgICAgICAgb25DaGFuZ2U9e29uQ2hhbmdlfVxuICAgICAgLz5cbiAgICApXG4gIH0sXG59KVxuXG5jb2x1bW5zLnNwbGljZShjb2x1bW5zLmxlbmd0aCAtMiwgMCx7XG4gIGtleTogJ3NlbGVjdGlvbicsXG4gIHdpZHRoOiA1MCxcbiAgXG4gIGNlbGxSZW5kZXJlcjogKHsgcm93RGF0YSB9KSA9PiB7XG4gICAgY29uc3Qgb25DaGFuZ2UgPSAodmFsdWU6IENoZWNrYm94VmFsdWVUeXBlKSA9PiAocm93RGF0YS5jaGVja2VkID0gdmFsdWUpXG4gICAgcmV0dXJuIDxTZWxlY3Rpb25DZWxsIHZhbHVlPXtyb3dEYXRhLmNoZWNrZWR9IG9uQ2hhbmdlPXtvbkNoYW5nZX0gLz5cbiAgfSxcblxuICBoZWFkZXJDZWxsUmVuZGVyZXI6ICgpID0+IHtcbiAgICBjb25zdCBfZGF0YSA9IHVucmVmKGRhdGEpXG4gICAgY29uc3Qgb25DaGFuZ2UgPSAodmFsdW)